<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>285</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>525</width>
    <height>285</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>525</width>
    <height>285</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Go To Setpoint</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources/nSOTScannerResources.qrc">
    <normaloff>:/nSOTScanner/Pictures/jj_schema.svg</normaloff>:/nSOTScanner/Pictures/jj_schema.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog{background-color: rgb(2, 2, 2);}
#line_11{color: rgb(168,168,168)}
#line_12{color: rgb(168,168,168)}
#line_13{color: rgb(168,168,168)}
#line_14{color: rgb(168,168,168)}
#line_15{color: rgb(168,168,168)}
#line_16{color: rgb(168,168,168)}
#line_17{color: rgb(168,168,168)}

QLabel{
color: rgb(168,168,168);
font: bold 10pt;
}</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>66</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#a8a8a8;&quot;&gt;Bias Setpoint:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>66</y>
     <width>96</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#a8a8a8;&quot;&gt;Field Setpoint:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gotoFieldBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>171</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:pressed#gotoFieldBtn{
color: rgb(95,107,166);
background-color:rgb(168,168,168);
border: 1px solid rgb(95,107,166);
border-radius: 5px
}

QPushButton#gotoFieldBtn{
color: rgb(95,107,166);
background-color:rgb(0,0,0);
border: 2px solid rgb(95,107,166);
border-radius: 5px
}</string>
   </property>
   <property name="text">
    <string>Go To 
Field</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>66</y>
     <width>19</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#a8a8a8;&quot;&gt;(T)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>66</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#a8a8a8;&quot;&gt;(V)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fieldRampRateLine">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>101</y>
     <width>45</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>101</y>
     <width>44</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#a8a8a8;&quot;&gt;(T/min)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_37">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>101</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#a8a8a8;&quot;&gt;Ramp Speed:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_38">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>136</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#a8a8a8;&quot;&gt;Step Delay:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="biasDelayLine">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>136</y>
     <width>45</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>136</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#a8a8a8;&quot;&gt;(ms)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="BiasInc">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>101</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#a8a8a8;&quot;&gt;Number of Steps:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="biasPntsLine">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>101</y>
     <width>45</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>500</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="blinkBtn">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>230</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:pressed#blinkBtn{
color: rgb(95,107,166);
background-color:rgb(168,168,168);
border: 1px solid rgb(95,107,166);
border-radius: 5px
}

QPushButton#blinkBtn{
color: rgb(95,107,166);
background-color:rgb(0,0,0);
border: 2px solid  rgb(95,107,166);
border-radius: 5px
}</string>
   </property>
   <property name="text">
    <string>Blink</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zeroFieldBtn">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>171</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:pressed#zeroFieldBtn{
color: rgb(95,107,166);
background-color:rgb(168,168,168);
border: 1px solid rgb(95,107,166);
border-radius: 5px
}

QPushButton#zeroFieldBtn{
color: rgb(95,107,166);
background-color:rgb(0,0,0);
border: 2px solid  rgb(95,107,166);
border-radius: 5px
}</string>
   </property>
   <property name="text">
    <string>Zero
Field</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fieldSetpntLine">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>66</y>
     <width>45</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="biasSetpntLine">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>66</y>
     <width>45</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>338</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; color:#a8a8a8;&quot;&gt;Magnet power supply and DC output channels are inherited from the characterization window&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="zeroBiasBtn">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>171</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:pressed#zeroBiasBtn{
color: rgb(95,107,166);
background-color:rgb(168,168,168);
border: 1px solid rgb(95,107,166);
border-radius: 5px
}

QPushButton#zeroBiasBtn{
color: rgb(95,107,166);
background-color:rgb(0,0,0);
border: 2px solid  rgb(95,107,166);
border-radius: 5px
}</string>
   </property>
   <property name="text">
    <string>Zero
Bias</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gotoBiasBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>171</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:pressed#gotoBiasBtn{
color: rgb(95,107,166);
background-color:rgb(168,168,168);
border: 1px solid rgb(95,107,166);
border-radius: 5px
}

QPushButton#gotoBiasBtn{
color: rgb(95,107,166);
background-color:rgb(0,0,0);
border: 2px solid rgb(95,107,166);
border-radius: 5px
}</string>
   </property>
   <property name="text">
    <string>Go To 
Bias</string>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>253</x>
     <y>52</y>
     <width>3</width>
     <height>140</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#line_8{color: rgb(168,168,168)}</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>51</y>
     <width>244</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#line_9{color: rgb(168,168,168);}</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>52</y>
     <width>3</width>
     <height>139</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#line_10{color: rgb(168,168,168)}</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_11">
   <property name="geometry">
    <rect>
     <x>513</x>
     <y>52</y>
     <width>3</width>
     <height>140</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#line_8{color: rgb(168,168,168)}</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_12">
   <property name="geometry">
    <rect>
     <x>271</x>
     <y>51</y>
     <width>244</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#line_9{color: rgb(168,168,168);}</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_13">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>52</y>
     <width>3</width>
     <height>139</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#line_13{color: rgb(168,168,168)}
#line_11{color: rgb(168,168,168)}
#line_12{color: rgb(168,168,168)}</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_14">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>190</y>
     <width>50</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#line_9{color: rgb(168,168,168);}</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_15">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>190</y>
     <width>50</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#line_9{color: rgb(168,168,168);}</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_16">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>190</y>
     <width>50</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#line_9{color: rgb(168,168,168);}</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_17">
   <property name="geometry">
    <rect>
     <x>271</x>
     <y>190</y>
     <width>50</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#line_9{color: rgb(168,168,168);}</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="currStatusLbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#a8a8a8;&quot;&gt;Status:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#a8a8a8;&quot;&gt;Go To Setpoint&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="currFieldLbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#a8a8a8;&quot;&gt;Current Field:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="currBiasLbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>296</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#a8a8a8;&quot;&gt;Current Bias:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_readSetpoint">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:pressed#push_readSetpoint{
color: rgb(95,107,166);
background-color:rgb(168,168,168);
border: 1px solid rgb(95,107,166);
border-radius: 5px
}

QPushButton#push_readSetpoint{
color: rgb(95,107,166);
background-color:rgb(0,0,0);
border: 2px solid  rgb(95,107,166);
border-radius: 5px
}</string>
   </property>
   <property name="text">
    <string>Read 
Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_Servers">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>15</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#push_Servers{
background: rgb(144, 140, 9);
 border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../Resources/nSOTScannerResources.qrc"/>
 </resources>
 <connections/>
</ui>
